src/pages/Dongwu/Login/Login.js

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
-
+import LoginArticle from './LoginArticle/LoginArticle';
+import LoginFooter from './LoginFooter/LoginFooter';
+import styles from './Login.module.scss';
```

Comment:
```suggestion
import './Login.scss';

```
- 일단 다른 동기분들과의 기술 스택도 맞춰야 하고 module은 커리큘럼상 없기 때문에 일단 그냥 SCSS 사용해주시는게 좋을거 같습니다.
- 가독성을 위해서 구분되는 요소들은 한줄 띄워주세요

Reply:
종택님, 해당 변경사항을 주말 이전에 적용해서 PR을 올리려 했으나, 예상치 못한 결과로 React 내에서 모든 className을 수정해주어야 하는 상황이 발생했습니다.

이에 해당 방식을 유지해도 될지 여쭤보고 싶습니다!